docs/user-manual/docinfo.html

```diff
@@ -0,0 +1,18 @@
+<script>
+document.addEventListener("DOMContentLoaded", function() {
+    if (!window.location.href.includes("latest")) {
```

Comment:
Probably worth gating this further so it only has an effect on the ActiveMQ website hsot where that dir name has meaning, and also that the "latest" check only actually considers the path rather than the full URL.